wp-content/themes/pub/wporg-learn-2020/archive-lesson-plan.php

```diff
@@ -15,16 +15,180 @@
+				<div class="lesson-plan-taxonomy-description"><?php echo esc_html__( 'How long a lesson is estimated to take.', 'wporg-learn' ); ?></div>
+				<ul class="list">
+					<?php
+					$durations = get_terms( 'duration', array(
+						'hide_empty' => false,
+						'orderby'    => 'name',
+						'order'      => 'ASC',
+					) );
+
+					$duration_index = 0;
+					$any_duration_arr = array();
+					foreach ( $durations as $duration ) :
+						if ( $duration_index < 3 ) :
+							?>
+					<li><a href="<?php echo esc_url( get_post_type_archive_link( 'lesson-plan' ) ) . '?duration[]=' . esc_html( $duration->term_id ); ?>"><?php echo esc_html( $duration->name ); ?><span class="dashicons dashicons-arrow-right-alt2"></span></a></li>
+							<?php
+						else :
+							$any_duration_arr[] = 'duration[]=' . $duration->term_id;
+						endif;
+						$duration_index++;
+					endforeach;
+
+					if ( $any_duration_arr ) :
+						$any_duration_html = implode( '&', $any_duration_arr );
+						?>
+						<li><a href="<?php echo esc_url( get_post_type_archive_link( 'lesson-plan' ) ) . '?' . esc_html( $any_duration_html ); ?>"><?php echo esc_html__( 'Any(over 60m)', 'wporg-learn' ); ?><span class="dashicons dashicons-arrow-right-alt2"></span></a></li>
+						<?php
+					endif;
+					?>
+				</ul>
+			</div>
+		</div>
+
+		<?php
+		$instruction_types = get_terms( 'instruction_type', array(
+			'hide_empty' => false,
+			'orderby' => 'id',
+		) );
+		?>
+		<div class="row lesson-plan-taxonomy last-grid">
+			<div class="card-grid card-grid_4">
+				<h2 class="h4 lesson-plan-taxonomy-header"><?php echo esc_html__( 'Format', 'wporg-learn' ); ?></h2>
+				<div class="lesson-plan-taxonomy-description"><?php echo esc_html__( 'Browse lesson plans based on their format.', 'wporg-learn' ); ?></div>
+				<?php foreach ( $instruction_types as $instruction_type ) : ?>
+				<div class="card">
+					<p class="taxonomy-title"><a href="<?php echo esc_url( get_post_type_archive_link( 'lesson-plan' ) ) . '?type[]=' . esc_html( $instruction_type->term_id ); ?>"><?php echo esc_html( $instruction_type->name ); ?></a></p>
+				</div>
+				<?php endforeach; ?>
+			</div>
+		</div>
+
+		<hr>
+
+		<div class="row align-middle around lesson-plan-cta">
+			<a href="<?php echo esc_url( get_post_type_archive_link( 'lesson-plan' ) . '?_view=all' ); ?>" class="button button-large button-secondary">
+				<?php esc_html_e( 'Browse all lesson plans', 'wporg-learn' ); ?>
+			</a>
+		</div>
+	</section>
+
+<?php else : ?>
 	<section>
 		<div class="row align-middle between section-heading section-heading--with-space">
-			<?php the_archive_title( '<h1 class="section-heading_title h2">', '</h1>' ); ?>
+			<?php the_archive_title( '<h1 class="section-heading_title bigger">', '</h1>' ); ?>
 			<?php get_template_part( 'template-parts/component', 'archive-search' ); ?>
 		</div>
 
 		<hr>
 
 		<div class="row gutters between">
-			<div class="card-grid col-9">
+			<div class="card-grid col-8 gap-64">
```

Comment:
This classname `gap-64` doesn't appear to follow a pattern used in this repo. I wonder if it might be better to add a class like `lp-list` to the parent div and add padding to that?

```
<div class="lp-list row gutters between">
```

```
.lp-list {
    padding-top: 4rem;
    padding-bottom: 0.25rem;
}
```

Reply:
@adamwoodnz I used `lp-container` to avoid `lp-list` conflicted.